Missionframework/scripts/fob_templates/unsung/template4.sqf

```diff
@@ -0,0 +1,164 @@
+    ["wx_chair", [-3.49, 1.43, 0.00], 322.25],
+    ["wx_chair", [-4.20, -0.30, 0.00], 283.18],
+    ["LAND_uns_vcshelter1", [-3.90, 0.16, 0.00], 0.00],
+    ["Land_Pod_Heli_Transport_04_fuel_F", [-9.75, 2.07, 0.00], 270.00],
+    ["Land_Pod_Heli_Transport_04_ammo_F", [-9.74, -2.72, 0.00], 270.00],
+    ["O_Truck_02_Ammo_F", [9.96, -1.73, 0.00], 270.00],
+    ["O_Truck_02_fuel_F", [10.24, 2.73, 0.00], 270.00],
+    ["O_MRAP_02_gmg_F", [9.04, -6.93, -0.00], 270.00],
+    ["Land_Pod_Heli_Transport_04_fuel_F", [-9.82, 6.30, 0.00], 270.00],
+    ["Land_Pod_Heli_Transport_04_ammo_F", [-10.12, -7.24, 0.00], 270.00],
+    ["LAND_uns_vcshelter2", [5.75, 9.72, 0.00], 90.00],
+    ["LAND_uns_vcshelter2", [9.24, 6.72, 0.00], 175.56],
+    ["Vil_Fence", [5.86, -13.88, 0.00], 0.00],
+    ["Vil_Fence", [14.74, 0.72, 0.00], 270.00],
+    ["uns_FlagCarrierMed", [-6.05, -13.43, 0.00], 0.00],
+    ["Vil_Fence", [-14.77, 0.42, 0.00], 270.00],
+    ["Vil_Fence", [-14.71, 6.41, 0.00], 270.00],
+    ["Vil_Fence", [14.79, 6.71, 0.00], 270.00],
+    ["uns_FlagCarrierNZ", [0.55, 14.33, 0.00], 0.00],
+    ["Vil_Fence", [6.21, 14.84, 0.00], 0.00],
+    ["Vil_Fence", [11.80, -13.94, 0.00], 0.00],
+    ["Vil_Fence", [14.65, -5.31, 0.00], 270.00],
+    ["Vil_Fence", [-6.50, -14.03, 0.00], 0.00],
+    ["Vil_Fence", [-14.86, -5.61, 0.00], 270.00],
+    ["Vil_Fence", [-6.15, 14.69, 0.00], 0.00],
+    ["Vil_Fence", [-14.66, 12.34, 0.00], 270.00],
+    ["Vil_Fence", [12.15, 14.78, 0.00], 0.00],
+    ["Vil_Fence", [14.85, 12.65, 0.00], 270.00],
+    ["LAND_uns_vctower1", [12.95, -12.88, 0.00], 318.28],
+    ["LAND_uns_vctower1", [-13.31, 13.96, 0.00], 144.79],
+    ["LAND_uns_vctower1", [-14.13, -13.21, 0.00], 42.90],
+    ["LAND_uns_vctower1", [13.68, 14.05, 0.00], 224.50],
+];
+
+_objectives_to_build = [
+    // Move all things which should be destroyed to accomplish the mission from the above to this array
+];
+
+_defenders_to_build = [
+    ["O_Soldier_F", [-4.09, 0.81, 0.00], 114.75],
+    ["O_Soldier_lite_F", [-4.64, -1.39, 0.00], 147.93],
+    ["O_Soldier_HAT_F", [6.97, 0.37, 0.00], 285.40],
+    ["O_Soldier_SL_F", [9.30, 8.28, 0.00], 250.30],
+    ["O_support_GMG_F", [1.15, -12.96, 0.00], 206.96],
+    ["O_Soldier_F", [-13.37, 0.92, 0.00], 271.85],
+    ["O_engineer_F", [12.94, -4.11, 0.00], 93.06],
+    ["O_medic_F", [8.98, 11.77, 0.00], 210.35],
+    ["O_Soldier_F", [-6.94, 13.46, 0.00], 0.00],
+    ["O_soldier_M_F", [13.41, -12.52, 4.86], 126.65],
+    ["O_Soldier_lite_F", [12.74, -13.21, 4.84], 197.32],
+    ["O_Soldier_F", [-13.78, 13.34, 4.87], 235.25],
+    ["O_HeavyGunner_F", [-13.55, -13.59, 4.87], 186.59],
+    ["O_Soldier_lite_F", [-14.38, -12.86, 4.84], 239.93],
+    ["O_soldier_M_F", [-13.29, 14.03, 4.85], 323.41],
+    ["O_soldier_M_F", [12.96, 14.41, 4.87], 0.00],
+    ["O_Soldier_F", [14.09, 13.37, 4.83], 98.03],
+];
+
+_base_corners = [
+    [40, 40, 0],
+    [40, -40, 0],
+    [-40, -40, 0],
+    [-40, 40, 0],
+];
```

Comment:
Recheck which one is the right one.
Remove unnecessary rpt stuff (clean the template)